raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -348,6 +356,62 @@ def next_transfer_pair(
     )
 
 
+def backward_transfer_pair(
+        backward_channel: NettingChannelState,
+        payer_transfer: LockedTransferSignedState,
+        pseudo_random_generator: random.Random,
+        block_number: typing.BlockNumber,
+) -> typing.Tuple[MediationPairState, typing.List[Event]]:
+    """ Sends a transfer backwards, allowing the previous hop to try a new
+    route.
+
+    When all the routes available for this node failed, send a transfer
+    backwards with the same amount and secrethash, allowing the previous hop to
+    do a retry.
+
+    Args:
+        backward_channel: The original channel which sent the mediated transfer
+            to this node.
+        payer_transfer: The *latest* payer transfer which is backing the
+            mediation.
+        block_number: The current block number.
+
+    Returns:
+        An empty list if there are not enough blocks to safely create a refund,
```

Comment:
nitpicking here, but the empty return value is rather
`(None, [])`
then an empty list, right?